README.md

```diff
@@ -2,23 +2,23 @@
 
 ## What is Olympus?
 
-Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset-backing and protocol owned value.
+Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset-backing and protocol-owned value. Read more about the game theory of Olympus [here](https://olympusdao.medium.com/the-game-theory-of-olympus-e4c5f19a77df).
 
 ## What is the point of Olympus?
 
 Our goal is to build a policy-controlled currency system, in which the behavior of the $OHM token is controlled at a high level by the DAO. In the long term, we believe this system can be used to optimize for stability and consistency so that $OHM can function as a global unit-of-account and medium-of-exchange currency. In the short term, we intend to optimize the system for growth and wealth creation.
 
 ## How do I participate in Olympus?
 
-There are two main strategies for market participants: staking and bonding. Stakers lock supply and receive a share of protocol profits as they are earned. Bonders provide locked liquidity and receive a fixed profit after a fixed period of time.
+There are two main strategies for market participants: staking and bonding. Stakers lock their $OHM tokens in return for the protocol profit, while bonders provide locked liquidity in exchange for $OHM tokens after a fixed period of time.
 
 Governance participants can get involved on our forum and through discussions on our discord. We are always looking for new community members to contribute!
 
 ## How can I benefit from Olympus?
 
-The main benefit for stakers comes from supply growth. The protocol takes a sizeable fee on all supply expansion, which is distributed primarily to stakers. Most gains for stakers will come from compounding balances, though price exposure remains an important consideration.
+The main benefit for stakers comes from supply growth. The protocol mints new $OHM tokens from the bond proceed, 90% of which are distributed to the stakers. Thus, the gain for stakers will come from their auto-compounding balances, though price exposure remains an important consideration. That is, if the increase in token balance outpaces the potential drop in price (due to inflation), stakers would make a profit.
```

Comment:
As per my understanding 90% of $OHM are staked but that is not equivalent to what is actually being distributed. Are we sure this explanation here is correct?

Reply:
90% from the profits from DAI bonds are distributed to stakers, but there are other sources of income as well

Reply:
I see. I was not aware of this. Since this page is the intro, should we prevent being too number focused and keep it simple? I can also see that these parameters change over time. Maybe we can limit more exact and technical documentation to one place so that we over time only have to keep one place updated.

Reply:
> 90% from the profits from DAI bonds are distributed to stakers, but there are other sources of income as well

I think the other sources of income here refer to the fees earned by the SLP and also Sushi earned from onsen pool

Reply:
> I see. I was not aware of this. Since this page is the intro, should we prevent being too number focused and keep it simple? I can also see that these parameters change over time. Maybe we can limit more exact and technical documentation to one place so that we over time only have to keep one place updated.

Ok, I will leave the number out